LiskHQ/lisk-sdk
# Update genesisBlock with interoperability assets

### What was the problem?

This PR resolves #8098 

### How was it solved?

- add data for ownChainDataSubstore and registeredNamesSubstore
- Update `chainID` to `04000000` and `tokenID` to `0400000000000000`

### How was it tested?

- Run app `./bin/run start --api-ipc --api-http --log debug`
- Call `interoperability_getOwnChainAccount` endpoint to see if we have relevant data
